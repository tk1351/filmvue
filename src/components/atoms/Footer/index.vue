<template>
  <footer class="footer">
    <strong>SNS</strong>
    <ul class="footer__list">
      <li>Twitter</li>
      <li>Filmarks</li>
    </ul>
    <small class="footer__owner">tk33220</small>
  </footer>
</template>

<style scoped lang="scss">
.footer {
  height: 160px;
  background-color: $main-color;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.footer__list {
  display: flex;
  gap: 40px;
}

.footer__owner {
  margin-top: 20px;
}
</style>
