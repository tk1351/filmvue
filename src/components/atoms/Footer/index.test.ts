import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Footer from './index.vue'

describe('Testing Footer Component', () => {
  it('正しくFooterコンポーネントが表示されていること', () => {
    const wrapper = mount(Footer)
    expect(wrapper.element).toMatchSnapshot()
  })
})
