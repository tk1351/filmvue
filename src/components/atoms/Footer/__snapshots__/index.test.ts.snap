// Vitest Snapshot v1

exports[`Testing Footer Component > 正しくFooterコンポーネントが表示されていること 1`] = `
<footer
  class="footer"
  data-v-e521488a=""
>
  <strong
    data-v-e521488a=""
  >
    SNS
  </strong>
  <ul
    class="footer__list"
    data-v-e521488a=""
  >
    <li
      data-v-e521488a=""
    >
      Twitter
    </li>
    <li
      data-v-e521488a=""
    >
      Filmarks
    </li>
  </ul>
  <small
    class="footer__owner"
    data-v-e521488a=""
  >
    tk33220
  </small>
</footer>
`;
